Duality/Resources/Font.cs

```diff
@@ -876,14 +890,15 @@ public int PickTextGlyph(string text, float x, float y)
 			return -1;
 		}
 
-		private void ProcessTextAdv(string text, int index, out GlyphData glyphData, out Rect uvRect, out float glyphXAdv, out float glyphXOff)
+		private void ProcessTextAdv(string text, int index, out GlyphData glyphData, out Rect uvRect, out float glyphXAdv, out float glyphXOff, out float glyphYOff)
 		{
 			char glyph = text[index];
 			int charIndex = (int)glyph > this.charLookup.Length ? 0 : this.charLookup[(int)glyph];
 			this.texture.LookupAtlas(charIndex, out uvRect);
 
 			this.GetGlyphData(glyph, out glyphData);
 			glyphXOff = -glyphData.OffsetX;
+			glyphYOff = glyphData.OffsetY;

```

Comment:
Interesting. I will definitely do this, but will have to test it some, since I'm not sure how it will impact the other calculations, especially kerning. Also I haven't seen this used in any of the generators for bitmap fonts.